### DISCUSSION TITLE
2FA

### DISCUSSION THREAD

**Discussion by snizzkers** [schwabe/ics-openvpn#1313] [Q&A] [answered]
Hello,
how can i configure 2FA? can't find any settings to do this.

Thank you very much for this great app.

**Comment by schwabe** [comment, marked as answer]
The app only support server initated 2FA/webauth by use of AUTH_PENDING messages. See https://github.com/OpenVPN/openvpn/blob/master/doc/management-notes.txt#L694 for details on the protocol. Since this is done dynamically there is no client configuration needed
